packages/client-fetch/src/utils.ts

```diff
@@ -440,6 +440,14 @@ class Interceptors<Interceptor> {
     this._fns = [];
   }
 
+  clear() {
+    this._fns = [];
+  }
+
+  exist(fn: Interceptor) {
```

Comment:
```suggestion
  exists(fn: Interceptor) {
```

Reply:
Sorry, I don’t understand. This is my first time making a PR in another project.. I obviously know what a changelog is, but I don’t know what you mean by making a changelog at the level of a PR. If you can explain it to me, I’ll do it. I’ve read something about changesets, and from what I see, it’s a tool to generate certain metadata about the changes and the package where they’re made. Is that what you’re referring to? And if so, how do I do it without submitting another PR?

Sorry for my ignorance 😞

Reply:
Yep, just create a new entry by running `pnpm changeset` and you can add a minor release for the fetch client. That way you'll get a proper credit in the release https://github.com/hey-api/openapi-ts/releases

Reply:
Hi again, I have just add changeset. Do I have to do somehing more ?

Reply:
Can you rename exist to exists? Is there a reason you called it exist?

Reply:
Even includes would work

Reply:
> Can you rename exist to exists? Is there a reason you called it exist?

Yes, no problema. I'd rename. The reason is a mistake with language

Reply:
I'm receiving some emails about fail tests.. Do I have do something?